packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <oc-table
+  <component
+    :is="determineViewMode"
```

Comment:
@kulmann @dschmidt @JanAckermann @JammingBen @lookacat looking for feedback here 🤓 

This is a primitive&naive first draft for switching the tiles view. The original idea was aiming for API-parity between the `<oc-table>` and a new `<oc-tiles>` component since a lot of potential shared logic (sorting, filtering, selectedFiles, context menu, sharing indicators, ...) live in the `<ResourceTable>` and would otherwise need duplication or abstraction. Do y'all see a smarter way? Because right now it feels like messy entanglement and I'm not sure if clean boundaries are possible^^

Reply:
@pascalwengerter for me it sounds good this way, or what makes you think it is messy this way? Not sure if there are better ideas

Reply:
At least i didn't have to think for long to understand how this is supposed to work ^^

Reply:
Just talked to @kulmann today, TL;DR is that this is the way to go (the other option would be adding lots of duplication which then would have to be abstracted away _again_)